lib/checks/media/caption.json

```diff
@@ -5,7 +5,6 @@
     "impact": "critical",
     "messages": {
       "pass": "The multimedia element has a captions track",
-      "fail": "The multimedia element does not have a captions track",
       "incomplete": "A captions track for this element could not be found"
```

Comment:
Could make this message more explicit about what to do to check whether an alternative exists